packages/plugin-mcp/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@payloadcms/plugin-mcp",
+  "version": "3.54.0",
+  "description": "MCP (Model Context Protocol) capabilities with Payload",
+  "keywords": [
+    "plugin",
+    "mcp",
+    "model context protocol"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/payloadcms/payload.git",
+    "directory": "packages/plugin-mcp"
+  },
+  "license": "MIT",
+  "author": "Payload <dev@payloadcms.com> (https://payloadcms.com)",
+  "maintainers": [
+    {
+      "name": "Payload",
+      "email": "info@payloadcms.com",
+      "url": "https://payloadcms.com"
+    }
+  ],
+  "sideEffects": false,
+  "type": "module",
+  "exports": {
+    ".": {
+      "import": "./src/index.ts",
+      "types": "./src/index.ts",
+      "default": "./src/index.ts"
+    }
+  },
+  "main": "./src/index.ts",
+  "types": "./src/index.ts",
+  "files": [
+    "src"
+  ],
+  "scripts": {
+    "build": "pnpm copyfiles && pnpm build:types && pnpm build:swc",
+    "build:swc": "swc ./src -d ./dist --config-file .swcrc --strip-leading-paths",
+    "build:types": "tsc --emitDeclarationOnly --outDir dist",
+    "clean": "rimraf {dist,*.tsbuildinfo}",
+    "copyfiles": "copyfiles -u 1 \"src/**/*.{html,css,scss,ttf,woff,woff2,eot,svg,jpg,png,json}\" dist/",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "pack:plugin": "pnpm prepublishOnly && pnpm copyfiles && pnpm pack",
+    "prepublishOnly": "pnpm clean && pnpm turbo build"
+  },
+  "dependencies": {
+    "@modelcontextprotocol/sdk": "^1.17.2",
+    "@vercel/mcp-adapter": "^1.0.0",
+    "zod": "^3.25.50"
```

Comment:
The Zod version is because of the MCP/sdk version. 
This will normalize soon when these supporting packages update their requirements.

`"@modelcontextprotocol/sdk": "^1.17.2"`
`"@vercel/mcp-adapter": "^1.0.0"`